content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -123,7 +123,7 @@ Due to technical constraints, the HorizontalPodAutoscaler controller
 cannot exactly determine the first time a pod becomes ready when
 determining whether to set aside certain CPU metrics. Instead, it
 considers a Pod "not yet ready" if it's unready and transitioned to
-unready within a short, configurable window of time since it started.
+ready within a short, configurable window of time since it started.
```

Comment:
I'm afraid I think the original wording is correct @pipo02mix 

I'd be happy to review pull requests that aim to clarify the existing behaviour, as the documentation is hard to follow here.

Reply:
I will try to come with a simpler phrase :D